metrics.ini

```diff
@@ -903,14 +903,16 @@ LineDancingCharacters="conf,ShowDancingCharacters"
 # Remove VideoRenderer if the player isn't on Windows because macOS and Linux only have one
 # valid VideoRenderer (opengl) where as players on Windows can choose (opengl or d3d).
 LineNames=(function() \
-   local lines = "VideoRenderer,1,2,3,RefreshRate,FSType,4,5,6,7,8,9,10,11,13,FNR,14,17,18,19,VisualDelaySeconds" \
+   local lines = "VideoRenderer,1,DisplayAspectRatio,DisplayResolution,RefreshRate,FSType,4,5,6,7,8,9,10,11,13,FNR,14,17,18,19,VisualDelaySeconds" \
    if type(ConfRefreshRate) ~= "function" then lines = lines:gsub("RefreshRate,", "") end \
    if type(ConfFullscreenType) ~= "function" then lines = lines:gsub("FSType,", "") end \
    if not HOOKS:GetArchName():lower():match("windows") then lines = lines:gsub("VideoRenderer","") end \
-   return lines \
+	 return lines \

```

Comment:
Looks like a tab got in there instead of spaces, and I don't know enough about Lua to tell if that's going to break stuff or not. Probably best to change it back to spaces, in any case. =)

Reply:
Done